src/thumbs.cpp

```diff
@@ -17,7 +17,7 @@
 namespace ddb{
 
 fs::path getThumbFromUserCache(const fs::path &imagePath, time_t modifiedTime, int thumbSize, bool forceRecreate){
-    if (std::rand() % 100 == 0) cleanupThumbsUserCache();
+    if (std::rand() % 1000 == 0) cleanupThumbsUserCache();
```

Comment:
How is this number calculated?

Reply:
Same (arbitrary). Suggestions to improve are welcome.